sentinelsat/products.py

```diff
@@ -37,16 +39,22 @@ def make_path_filter(pattern, exclude=False):
 
     Parameters
     ----------
-    pattern : str
-        glob patter for files selection
+    pattern : str, sequence[str]
+        glob patter (or sequence of patterns) for files selection
     exclude : bool, optional
         if set to True then files matching the specified pattern are excluded. Default False.
-
-    .. versionadded:: 0.15
+       The `pattern` parameter can also be a list of patterns.
```

Comment:
@j08lue `chain_filters` allows to chain multiple filters. It can help to add path filter along with a size filter, as @avalentino mentioned in the thread #574.

I will definitely try for adding a test, and update the line and omit the new line.

Reply:
I have updated the code as per suggestions, will work on including test cases.